pkg/kv/kvserver/kvflowcontrol/rac2/range_controller.go

```diff
@@ -1886,34 +1898,41 @@ type replicaSendStream struct {
 		}
 		closed bool
 	}
+	// tracker contains entries that have been sent, and have had send-tokens
+	// deducted (and will have had eval-tokens deducted iff index >=
+	// nextRaftIndexInitial).
+	//
+	// Contains no entries in probeRecentlyNoSendQ.
+	tracker Tracker
```

Comment:
Is the reason why it's safe to move `tracker` out of `replicaSendStream.mu` the fact that it has its own mutex?

With this change, is there a mental model for what other mutexes are held when `tracker` is accessed? E.g. something like: `raftMu` is held when tokens are registered, and not held when deregistered; or maybe `raftMu` is always held when mutating it?

If there is an easy was to describe this, consider putting it into the comment. If `tracker` is always accessed under `raftMu`, consider making a `raftMu struct` section here containing it.